runtime/ms_rest_azure/spec/polling_state_spec.rb

```diff
@@ -10,25 +10,31 @@ module MsRestAzure
   describe PollingState do
     it 'should initialize status from flattened response body' do
       response_body = double('response_body', :provisioning_state => 'InProgress')
-      response = double('response',
```

Comment:
**was the test before these changes valid? By this question I mean, would it be valid to have a nil "response" with a response body?**
No. A nil response is not a valid response. Think of the scenario that you call a web service. It should return something with some status. Else, it is a timeout